### PR TITLE
[zh-cn]: update translation of `HTTP` List of default Accept values

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md [hunk 1]
```diff
@@ -1,80 +1,91 @@
...
-| Opera               | `text/html, application/xml;q=0.9, application/xhtml+xml, image/png, image/webp, image/jpeg, image/gif, image/x-xbitmap, */*;q=0.1`                                     |                                                                                                                                                   |
+| 用户代理                  | 值                                                                                                                                  |
+| ------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| Firefox 92 及更高版本 [1] | `text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,*/*;q=0.8`                                             |
+| Firefox 72 至 91 [1]      | `text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8`                                                        |
```
Comment (T34-active) [inline files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md:19]:
```suggestion
| Firefox 72 至 91 版本 [1]      | `text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8`                                                        |
```

File: files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md [hunk 2]
```diff
@@ -1,80 +1,91 @@
...
+| Firefox 66 至 71 [1]      | `text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8`                                                                   |
```
Comment (T34-active) [inline files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md:20]:
```suggestion
| Firefox 66 至 71 版本 [1]      | `text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8`                                                                   |
```

File: files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md [hunk 3]
```diff
@@ -1,80 +1,91 @@
...
-| Internet Explorer 8 及更早版本 | `*/*`                                                                 | 请参见 [IE and the Accept Header (IEInternals' MSDN blog)](http://blogs.msdn.com/b/ieinternals/archive/2009/07/01/ie-and-the-accept-header.aspx)                                 |
-| Internet Explorer 9            | `image/png,image/svg+xml,image/*;q=0.8, */*;q=0.5`                    | 请参见 [Fiddler is better with Internet Explorer 9 (IEInternals' MSDN blog)](http://blogs.msdn.com/b/fiddler/archive/2011/02/10/fiddler-is-better-with-internet-explorer-9.aspx) |
+| 用户代理                           | 值                                                                         |
+| ---------------------------------- | -------------------------------------------------------------------------- |
+| Firefox 92 及更高版本 [1]          | `image/avif,image/webp,*/*`                                                |
+| Firefox 65 至 91 [1]               | `image/webp,*/*`                                                           |
```
Comment (T34-active) [inline files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md:39]:
```suggestion
| Firefox 65 至 91 版本 [1]               | `image/webp,*/*`                                                           |
```

File: files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md [hunk 4]
```diff
@@ -1,80 +1,91 @@
...
+| Firefox 47 至 63 [1]               | `*/*`                                                                      |
```
Comment (T34-active) [inline files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md:40]:
```suggestion
| Firefox 47 至 63 版本 [1]               | `*/*`                                                                      |
```

File: files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md [hunk 5]
```diff
@@ -1,80 +1,91 @@
...
+| Firefox 47 之前 [1]                | `image/png,image/*;q=0.8,*/*;q=0.5`                                        |
```
Comment (T34-active) [inline files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md:41]:
```suggestion
| Firefox 47 之前版本 [1]                | `image/png,image/*;q=0.8,*/*;q=0.5`                                        |
```

**Review by T34-active — COMMENTED** [review]
fix content

File: files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md [hunk 6]
```diff
@@ -1,80 +1,91 @@
...
+| Safari（自 Mac OS Big Sur 版本起） | `image/webp,image/png,image/svg+xml,image/*;q=0.8,video/*;q=0.8,*/*;q=0.5` |
+| Safari（Mac OS Big Sur 之前版本）  | `image/png,image/svg+xml,image/*;q=0.8,video/*;q=0.8,*/*;q=0.5`            |
+| Chrome 和 Edge 121 及更高版本      | `image/avif,image/webp,image/apng,image/*,*/*;q=0.8`                       |
+
+\[1] 可以使用 `image.http.accept` 参数（_[source](https://searchfox.org/mozilla-central/search?q=image.http.accept)_）修改此值。
 
-## video 请求
+## video 的值
 
 通过 HTML {{HTMLElement("video")}} 元素请求一个 video 时，大多数浏览器会使用特定值。
```
Comment (yin1999) [inline files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md:50]:
```suggestion
通过 HTML {{HTMLElement("video")}} 元素请求一个视频时，大多数浏览器会使用特定值。
```

File: files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md [hunk 7]
```diff
@@ -1,80 +1,91 @@
...
+| 用户代理               | 值                                                                                 |
+| ---------------------- | ---------------------------------------------------------------------------------- |
+| Firefox 3.6 及更高版本 | `video/webm,video/ogg,video/*;q=0.9,application/ogg;q=0.7,audio/*;q=0.6,*/*;q=0.5` |
+| Firefox 3.6 之前的版本 | _不支持 {{HTMLElement("video")}}_                                                  |
+| Chrome                 | `*/*`                                                                              |
 
-## audio 请求
+## audio 资源的值
 
 通过 HTML {{HTMLElement("audio")}} 元素请求 audio 资源时，大多数浏览器会使用特定值。
```
Comment (yin1999) [inline files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md:60]:
```suggestion
通过 HTML {{HTMLElement("audio")}} 元素请求音频资源时，大多数浏览器会使用特定值。
```

File: files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md [hunk 8]
```diff
@@ -1,80 +1,91 @@
...
+| -------------------------- | -------------------------------------------------------------------------------------------- |
+| Firefox 3.6 及更高版本 [1] | `audio/webm,audio/ogg,audio/wav,audio/*;q=0.9,application/ogg;q=0.7,video/*;q=0.6,*/*;q=0.5` |
+| Safari、Chrome             | `*/*`                                                                                        |
 
-## scripts 请求
+\[1] 参见 [bug 489071](https://bugzil.la/489071)。
+
+## script 的值
 
 当通过 {{HTMLElement("script")}} 元素请求 script 时，一些浏览器使用特定值。
```
Comment (yin1999) [inline files/zh-cn/web/http/content_negotiation/list_of_default_accept_values/index.md:71]:
```suggestion
当通过 {{HTMLElement("script")}} 元素请求脚本时，一些浏览器使用特定值。
```

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**